Boo.rip/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "author": {
+      "name": "Dominik",
+      "id": "594491170248458250"
+    },
+    "service": "Boo.rip",
+    "description": {
+      "en": "Boo is a elite connection optimizer that aims to improve user gameplay."
+    },
+    "url": "boo.rip",
+    "version": "0.1",
+    "logo": "https://cdn.discordapp.com/attachments/660924113308876820/660924316795404323/boo-official.png",
```

Comment:
Please upload the image somewhere else, dont use discord's cdn

Reply:
Done